### PR TITLE
fix pipeline doc

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
add `kubectl create ns` commend before needed. And replace example of custom task. The current custom task will be used later in the usage example.
**Which issue(s) this PR fixes**:
a part of #435

### CODE REVIEW

File: docs/content/en/docs/pipeline/setting.md [hunk 1]
```diff
@@ -123,15 +125,12 @@ A Custom Task allows for greater flexibility by defining tasks directly within t
...
-        - '/bin/sh'
-        - '-c'
+        - /bin/sh
+        - -c
       args:
-        - 'echo Hello World'
-      env:
-        - name: EXAMPLE_ENV
-          value: 'example'
+        - "cat $(workspaces.source.path)/README.md"' | kubectl apply -f -
```
Comment (Xieql) [inline docs/content/en/docs/pipeline/setting.md:135]: `| kubectl apply -f - ` ?
why add this?

File: docs/content/en/docs/pipeline/operation.md [hunk 2]
```diff
@@ -101,12 +100,10 @@ el-test-predefined-task-listener-7d5774d7c-82whm   1/1     Running   0
 
 ## Exposing Services and Webhook Settings
 
-Within the Kubernetes cluster, both `el-test-custom-task-listener` and `el-test-predefined-task-listener` are configured as ClusterIP services, 
-meaning they are only accessible internally. 
-To enable external systems, like webhooks, to access these services, we need to expose them outside the cluster. 
+Within the Kubernetes cluster, both `el-test-custom-task-listener` and `el-test-predefined-task-listener` are configured as ClusterIP services, meaning they are only accessible internally.
```
Comment (Xieql) [inline docs/content/en/docs/pipeline/operation.md:103]: Why merge the original content? For code reading, a line that is too long is difficult to read.

File: docs/content/en/docs/pipeline/operation.md [hunk 3]
```diff
@@ -101,12 +100,10 @@ el-test-predefined-task-listener-7d5774d7c-82whm   1/1     Running   0
...
+To enable external systems, like webhooks, to access these services, we need to expose them outside the cluster.
 
-For demonstration and testing convenience, a straightforward method for this is by using the `kubectl port-forward` command, 
-which allows us to forward a port from an internal service to a local server or computer.
+For demonstration and testing convenience, a straightforward method for this is by using the `kubectl port-forward` command, which allows us to forward a port from an internal service to a local server or computer.
```
Comment (Xieql) [inline docs/content/en/docs/pipeline/operation.md:106]: same

File: docs/content/en/docs/pipeline/setting.md [hunk 4]
```diff
@@ -123,15 +125,12 @@ A Custom Task allows for greater flexibility by defining tasks directly within t
 
 ```yaml
 tasks:
-  - name: custom-task-example
+  - name: cat-readme
```
Comment (Xieql) [inline docs/content/en/docs/pipeline/setting.md:128]: this just a example

File: docs/content/en/docs/pipeline/setting.md [hunk 5]
```diff
@@ -123,15 +125,12 @@ A Custom Task allows for greater flexibility by defining tasks directly within t
...
     customTask:
-      image: 'example-image'
+      image: zshusers/zsh:4.3.15
```
Comment (Xieql) [inline docs/content/en/docs/pipeline/setting.md:130]: this just a example

File: docs/content/en/docs/pipeline/setting.md [hunk 6]
```diff
@@ -123,15 +125,12 @@ A Custom Task allows for greater flexibility by defining tasks directly within t
...
       command:
-        - '/bin/sh'
-        - '-c'
+        - /bin/sh
+        - -c
```
Comment (Xieql) [inline docs/content/en/docs/pipeline/setting.md:133]: You don't need to change this.

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**